src/neuroconv/utils/json_schema.py

```diff
@@ -3,8 +3,11 @@
 import json
 import inspect
 from datetime import datetime
+from typing import _LiteralGenericAlias
```

Comment:
I should have known importing a private class would bite me in the butt. `typing._LiteralGenericAlias` isn't available in Python 3.8. I'll need to work out a more robust solution.

Reply:
Was this resolved? All tests are green after all